CHANGELOG.md

```diff
@@ -8,9 +8,12 @@ have notable changes.
 
 Changes since v2.19
 
-## Fixes
+### Fixes
 - Show the selected but disabled resources in Change resources autocomplete (#2702, #2693)
 
+### Additions
+- "More info" support for DOI style resources (when :enable-doi flag is set) (#2701)
```

Comment:
Here in the Markdown we could use the backticks like `:enable-doi` to make the flag stand out.